src/builder.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fastify = require('fastify')()
+const got = require('got');
+//USELESS
+const fastJson = require('fast-json-stringify')
+
+const paramSchema = {
+    type: 'object',
+    properties: {
+        'number': { type: 'integer' }
+      },
+      required: ['number']
+}
+const schema = {
+    params: paramSchema
+}
+
+function Builder(){
+
+    fastify.get('/', async (request, reply) => {
+        reply.send({statusCode: 200, msg: "Hello world!"})
+    } )
+    
+    fastify.get('/riders', async (request, reply) => {
+        try {
+            console.log("BEFORE GOT")
+            const response = await got('https://demo.test.mia-platform.eu/v2/riders/', { json: true});
```

Comment:
Use an environment variable for the CRUD url so that we do not leak our projects urls; this endpoint is fake and for demos but it still sensitive